fearless/Modules/NewWallet/ChainAccount/ChainAccountPresenter.swift

```diff
@@ -115,3 +150,9 @@ extension ChainAccountPresenter: Localizable {
         provideViewModel()
     }
 }
+
+extension ChainAccountPresenter: ModalPickerViewControllerDelegate {
+    func modalPickerDidSelectModelAtIndex(_ index: Int, context _: AnyObject?) {
+        wireframe.presentPurchaseWebView(from: view, action: getPurchaseActions()[index])
```

Comment:
what is context here? maybe it will be safer to use it instead of getPurchaseActions(). in case order inside modal picker will be changed, current logic will not handle that case correctly

Reply:
I have no idea, to be honest. Context never used for no one implementation of that method

Reply:
It always nil and seems like do nothing